### ISSUE TITLE
There's no use of Pizza in the code.

### ISSUE THREAD

**Issue by briandfoy** [golang/tour#611]
Context: http://127.0.0.1:3999/basics/3

I don't think you'll lose any meaning by not mentioning Pizza at all.

**Comment by bgadrian** [comment]
I agree, and it also confuses the devs because the pizza property does not exists in the math package.

**Comment by briandfoy** [comment]
Closing this just so it's not in my list of open things.
